### DISCUSSION TITLE
Are listed accelerators still active?

### DISCUSSION THREAD

**Discussion by jeffmartens** [portland-startups/docs#91] [General]
- Cambia link goes to a 404
- e1ectr0n has no info on its site about an accelerator-
- Founders’ Pad doesn't have a website
- JLR doesn't accelerator/incubate anymore
- Is PIE still operating as PIE, or is it an Autodesk thing now?

**Comment by dbarcos-formos** [comment]
I’d love to get North Bank Innovations on the list as a support org. We are ramping up our event schedule and listings.

[www.northbankinnovations.org](http://www.northbankinnovations.org/)
[www.meetup.com/northbankinno](http://www.meetup.com/northbankinno)
